modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2144,9 +2149,21 @@
 
 
 	function m.includePath(cfg)
```

Comment:
Those 2 methods should probably be merged, as they both handle cfg.externalincludedirs.
or vs2022_vcxproj created?